src/app/admin/admin-reports/filtered-collections/filtered-collections.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="container">
+    <div class="metadata-registry row">
+        <div class="col-12">
+
+            <h2 id="header" class="border-bottom pb-2">{{ "admin.reports.collections.head" | translate }}</h2>
```

Comment:
For accessibility, this should be an `<h1>`.  All pages should start with an `<h1>`

Reply:
Done.